Lib/test/support/threading_helper.py

```diff
@@ -0,0 +1,148 @@
+import contextlib
+import functools
+import _thread
+import threading
+import time
+
+from test import support
+
```

Comment:
it's the opportunity to make the code a little bit more PEP 8 compliant: please add an empty line here :-D

Reply:
got it, thanks.